xbmc/video/VideoInfoScanner.cpp

```diff
@@ -2029,7 +2029,7 @@ namespace VIDEO
       CGUIDialogOK::ShowAndGetInput(20448,20449,20022,20022);
       return false;
     }
-    return CGUIDialogYesNo::ShowAndGetInput(20448,20449,20450,20022);
+    return CGUIDialogYesNo::ShowAndGetInput(20448,20449);
```

Comment:
oh, and the yes/no dialog requires 4 parameters, so you have to replace the removed ones with -1 here